schema/meta_schema_language_java.yaml

```diff
@@ -304,18 +292,14 @@ typeSupportStatuses:
       - property: swift
         status: not_applicable
   - type: ExperimentalJaegerRemoteSampler
-    status: ignored
+    status: supported
```

Comment:
Pending updates in https://github.com/open-telemetry/opentelemetry-java/pull/7943